prompts.md

```
@@ -16,103 +16,666 @@ Puedes añadir adicionalmente la conversación completa como link o archivo adju
+
+Base de datos: PostgreSQL.
+
+Autenticación: JWT (con opción de OAuth para Google en fases posteriores).
+
+Entorno de desarrollo: Docker para uniformizar backend + base de datos.
+
+Pasos de instalación (modo local)
+
+Clonar el repositorio
+
+git clone https://github.com/usuario/proyecto-mtg-market.git
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)